static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -31,12 +34,18 @@ const EVENTS = [
 ];
 const gaTracker = config.googleAnalytics.trackers.editorial;
 
+export type MediaEventT = {
```

Comment:
I want to have a type other than `Object`, but this seems really clunky and the `T` suffix is not pretty. Could go all .net and give it an `I` prefix instead.